### DISCUSSION TITLE
Invalid expression identified, How to resolve this issue on MTex?

### DISCUSSION THREAD

**Discussion by Charbak22** [mtex-toolbox/mtex#2286] [Ask Anything]
![Screenshot 2024-12-22 174431](https://github.com/user-attachments/assets/d61ba7b5-fa65-4604-84ac-35018c7f6db0)
This issue arises when I copy this code from the MTex Documentation, run it on Matlab. How to resolve this issue?

**Comment by kilir** [comment]
Hi,
this is not Matlab code but mtex output you were trying to execute.
Cheers,
Rüdiger
